packages/entity-browser/src/components/DetailView/DetailView.js

```diff
@@ -13,6 +13,8 @@ const DetailView = props => {
         submitForm={props.submitForm}
         formDefinition={props.formDefinition}
         entity={props.entity}
+        loadRelationEntities={props.loadRelationEntities}
```

Comment:
I think the name is ok because the function itself just loads the related entities. In the DetailForm this function is passed to the onFocus function which gets called if someone clicks on the field.